src/App.tsx

```diff
@@ -2,18 +2,16 @@ import React, { useContext, useEffect, useState } from "react";
 import { Link, Route, Switch, useLocation, Redirect } from "react-router-dom";
 import './App.css';
 import Icon from './assets/images/two-tone-light.svg';
-import Data from "./components/pages/Data";
+import Dashboard from "./components/pages/Data";
 import About from './components/static/About';
 import Auth from './components/static/Auth/Auth';
-import Home from './components/static/Home';
+import Data from './components/static/Home';
```

Comment:
So as I understand, what was formerly called the Home tab will now be called the Data tab right? Is this something Rahul is expecting or nah?

Reply:
yep! discussed